site/docs/v1/tech/core-concepts/applications.adoc

```diff
@@ -21,6 +22,15 @@ Here's a brief video covering some aspects of applications:
 
 video::DaZbwrA7M90[youtube,width=560,height=315]
 
+== Core Concepts Relationships
```

Comment:
Thanks. I think it's better to show these because sometimes folks are confused about the relationship. But figured out a way to bold each entity on the page discussing them. Does that help?

Reply:
Absolutely.  Makes much more sense now.